sim/core/consumes.go

```diff
@@ -17,6 +17,37 @@ func applyConsumeEffects(agent Agent) {
 
 	if consumes.Flask != proto.Flask_FlaskUnknown {
 		switch consumes.Flask {
+		case proto.Flask_FlaskOfTitanicStrength:
+			character.AddStats(stats.Stats{
+				stats.Strength: 300,
+			})
+			if character.HasProfession(proto.Profession_Alchemy) {
+				character.AddStats(stats.Stats{
+					stats.Strength: 80,
+				})
+			}
```

Comment:
Its prefered if we do a single AddStats instead of 2 so better to change all the alchemy bonuses to be added to a variable and then a single add stats with the total ammount

Reply:
Fixed for the new flasks, let the old ones be there because i dont know the logic for them, and they seems to have different values for each one